layers/ubuntu1604/go/BUILD

```diff
@@ -34,7 +34,7 @@ language_tool_layer(
     base = "@ubuntu1604//image",
     env = {
         "GOPATH": "/go",
-        "PATH": "$PATH:/usr/local/go/bin",
+        "PATH": "$$PATH:/usr/local/go/bin",
```

Comment:
Is this a new Bazel thing?

Reply:
yes, it's the supportrd way of doing this now, I double checked the test that verifies this is still correctly verifying the expected path